examples/quadrotor.py

```diff
@@ -19,12 +18,15 @@
 target_quat = pinocchio.Quaternion(1, 0, 0, 0)
 
 state = crocoddyl.StateMultibody(robot_model)
-distance_rotor_COG = 0.1525
+d_cog = 0.1525
 cf = 6.6e-5
 cm = 1e-6
 u_lim = 5
 l_lim = 0.1
-actModel = ActuationModelUAM(state, '+', distance_rotor_COG, cm, cf, u_lim, l_lim)
+tau_f = np.matrix([[0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0], [1.0, 1.0, 1.0, 1.0], [0.0, d_cog, 0.0, -d_cog],
+                   [-d_cog, 0.0, d_cog, 0.0], [-cm / cf, cm / cf, -cm / cf, cm / cf]])
+
+actModel = crocoddyl.ActuationModelMultiCopterBase(state, 4, tau_f)
```

Comment:
cf, cm are aerodynamic parameters of the propellers and d_cog is the distance of the respective propellers to the center of gravity.

To get the matrix tau_f from the stated parameters, it is not only necessary their values but you also need to know how the propellers are distributed around the center of gravity (number of propellers and rotation direction). It would be great to have this info in the urdf and you can extract it from there. However, I don't think this part has to be done in Crocoddyl. The essential part you need to know is the tau_f, that is why I only added this to the API.

In that sense, I have a separated library that parses a yaml file where you can specify a multirotor geometry and outputs the tau_f matrix that Crocoddyl needs.

Reply:
Ok, I didn't think about different rotor geometries.

Then it makes sense to me, as well, to keep the API with `tau_f`. Let's keep it as it is --> no action here.

Thank you for the explanation.